Add CW23 blog (#630)
open-life-science/open-life-science.github.io: _posts/2023-06-28-OLS-at-CollabW23.md

Together with Mayya, I facilitated a session on self care (slides can be found on [Zenodo](https://doi.org/10.5281/zenodo.8060439)). 
I was the in-person facilitator for our session while Mayya led the discussion with our online attendees. 
As we had technical issues during our session, this was a great way of co- facilitating as Mayya could make sure remote participants are looked after while I kept a conversation in the room going until our technical issues were resolved.

SSI/OLS Fellow Reina Camacho Toro led a session exploring if current concepts and practices around Open Science are global and universal. 
She and her co-facilitators piloted a survey aiming to understand the current barriers, successful practices and possible improvements needed for the implementation of Open Science in Latin America.

I also attended a session on (agile) project management and a workshop playing through an evil collaboration scenario. 
It is great to see these topics being so central in a conference as they are usually seen as low priority and getting in the way of getting work done. 

## OLS community activities and highlights

> 

Reviewer: It might be good to have an introduction sentence to mention that now different people will share their experience. It is not really clear for me from the title

Author: Good point - added something now!

I also had the pleasure of sharing one of the workshop sessions with Aman. 
In this session, I presented the pipeline we are developing at [Ersilia](https://ersilia.io) to facilitate the onboarding of new Open Source contributors (mainly from the Outreachy program) to our ecosystem, very much based on the learnings from our participation in the OLS cohort 5. 

### Saranjeet Kaur Bhogal
I had the pleasure of interacting with Malvika Sharan on multiple occasions. 
During the Hack Day, I had the opportunity to collaborate with Gemma Turon (and others) on a project focusing on Software Authorship. 
It was a rewarding experience to work alongside Gemma and bring our collective expertise to the table. 
In addition to that, during the Collaborative Ideas session, I had the opportunity of collaborating with Aman Goel (and others) to come up with the idea of creating a RSE Kaleidoscope. 
It was inspiring to see our vision taking shape as we shared perspectives.

### Aman Goel

> This year’s Collaboration Workshop has been undoubtedly the highlight of my year so far. 

Reviewer: ```suggestion
This year's Collaboration Workshop has been undoubtedly the highlight of my year so far. 
```

Having met so much of the community online via OLS and SSI, it was quite refreshing to finally meet Zoom-familiar folks in person! 
Right from the event registration until the last minute of the hack-day, I had the opportunity to closely interact and collaborate with fellow OLS community members. 
Moreover, I had the pleasure of introducing OLS to several SSI community members at the workshop, and their enthusiasm to learn more about the mentorship program was truly inspiring. 

> As Gemma mentioned, I shared a workshop session with her, where I showcased my OLS project, titled ‘[The Undergraduate's Guide To Research Software Engineering](https://software.ac.uk/cw23/mini-workshops-and-demo-sessions#3.4b)’, which received a warm reception from the attendees. 

Reviewer: ```suggestion
As Gemma mentioned, I shared a workshop session with her, where I showcased my OLS project, titled ["The Undergraduate's Guide To Research Software Engineering"](https://software.ac.uk/cw23/mini-workshops-and-demo-sessions#3.4b), which received a warm reception from the attendees. 
```

> All in all, the workshop was both productive and rejuvenating, and I’m looking forward to similar events in the future.

Reviewer: ```suggestion
All in all, the workshop was both productive and rejuvenating, and I'm looking forward to similar events in the future.
```
